src/monika/Array.java

```diff
@@ -0,0 +1,96 @@
+
+//write a single program having different methods.
+ 
+//a) To find Min number from given array
+//b) To find Max number from given array
+//c) To find Avg of all numbers from give array
+//d) To display all numbers [0 to length]
+//e) To display all numbers [length to 0]
+//g) Method to find average of min and max number from array.
+
+package monika;
+public class Array {
+
+	  int getMaxNumber(int num[]) {
+
+		int max = num[0];
+		for(int index=0; index < num.length; index++) {
+
+			if(max < num[index])
+				max = num[index];
+		}
+
+		return max;
+	 }
+
+	  int getMinNumber(int num1[]) {
+
+			int min =num1[0];
+			for(int index=0; index < num1.length; index++) {
+
+				if(min > num1[index])
+					min = num1[index];
+
+			}
+			return min;
+		}
+
+	  double getAvgNumbers(int num2[]) {
+
+		   double avg = 0;
+		   for(int index=0; index < num2.length; index++) {
+
+			   avg = avg + num2[index]/5;
+
+		   }
+				return avg;
+		}
+
+	  void getLengthofArray(int num3[]) {
+
+		   for(int index=0; index < num3.length; index++) {
+
+				System.out.println(num3[index]);
+
+		   }	
+
+		}
+
+       void getNumbersInReverse(int num4[]) {
+
+		   for(int index= num4.length-1; index >= 0; index--) {
+
+					System.out.println(num4[index]);
+				}
+
+			}
+
+        double getMinMaxAvg(int maxNum, int minNum) {
+
+            double avgOfMinMax = (maxNum + minNum)/2;
```

Comment:
I think we have requirement where you need to find max & min number then you need to find avg of both, user will give us array, and you need to find min & max value. and then avg of both, I hope you got my point.


You can write code like this, I remove variable and directly doing sum of min & max, no problem, if you are not comfortable to write code like below, just optimize.

int avgOfMinMax = (min(input)+max(input))/2